stetho/src/main/java/com/facebook/stetho/inspector/elements/android/AndroidDocumentProvider.java

```diff
@@ -71,15 +72,17 @@ public void run() {
     }
   };
 
-  public AndroidDOMProvider(Application application) {
+  public AndroidDocumentProvider(Application application, Handler handler) {
```

Comment:
why not just have the 2nd parameter be a `ThreadBound` ? we don't use the handler; we only used it to implement `ThreadBound`. But the factory does that now and we're deferring to it ... so why not have our ctor here take a `ThreadBound`, pass it to the super class, and don't even bother storing it in a field? That will also allow us to remove the ctor of `ThreadBoundProxy` which takes a `Handler`, and then we can remove the `HandlerThreadBoundWrapper`. (this is what I intended when I made earlier comments about this :) )